api/documents/DMStore.ts

```diff
@@ -0,0 +1,79 @@
+import {AxiosResponse} from 'axios'
+import * as FormData from 'form-data'
+import {Fields, File, Files} from 'formidable'
+import * as fs from 'fs'
+import * as stream from 'stream'
+import { config } from '../config'
+import { http } from '../lib/http'
+import * as log4jui from '../lib/log4jui'
+import {JUILogger} from "../lib/models"
+import { asyncReturnOrError } from '../lib/util'
+import {DMDocument, DMDocuments} from './document.interface'
+
+const url: string = config.services.documents.api
+
+const logger: JUILogger = log4jui.getLogger('dm-store')
```

Comment:
As discussed please update to use log.tracker using the base method in our interceptor class as our logging utility moving forward.  thanks.

Reply:
the interceptor picks up all requests so will pick up for document upload/view.